Getting-Started.md

```diff
@@ -39,15 +39,14 @@ Step 3: Ensuring your RWS meets the technical requirements
 Step 4: Testing your RWS locally
 --------------------------------
 
-Once you've made your changes to your local branch, you can open a terminal and run the command `python3 check_sites.py`. Currently, this will produce a number of check failures due to older submissions in the set failing recent changes to checks. If no members of your set are mentioned in the list of failed checks, then your submission should be fine. If you would like a cleaner way of looking for problems in your set, follow these steps:
+Once you've made your changes to your local branch, you can open a terminal and run the command 
 
-1.  Run `cp related_website_sets.JSON my_rws.JSON` in your shell
+`python3 check_sites.py --testing_sets=<your primary site>`
 
-2.  Make your changes to `my_rws.JSON` and make sure `related_website_sets.JSON` is identical to the version in main.
+You will get the results of any failed tests in the terminal. Otherwise, you will see "success" if your changes are passing all of the checks. Make sure that the text of the primary site you're passing into the command line is identical to the primary site you have listed in the related_website_sets.JSON file, or the tests will fail. If you would like to test multiple Related Website Sets at once, you can run the command above with multiple primaries in a comma separated list, like so:
 
-3.  To test your local changes, run `python3 check_sites.py -i my_rws.JSON --with_diff`. When this command finishes running, you will either see "success" (meaning your submission passed all of the checks) or you will see a list of failed checks.
 
-4.  When you are ready to submit, copy your changes from `my_rws.JSON` into `related_website_sets.JSON` and delete `my_rws.JSON`
+`python3 check_sites.py --testing_sets=<primary 1>,<primary 2>,<etc.>`
```

Comment:
```suggestion
`python3 check_sites.py --testing_sets="https://foo.example,https://bar.example"`
```

I think it would be helpful to show a more concrete example, so that people don't have to guess whether we require `https://`, whether quotes are necessary, etc.